views/address.tmpl

```diff
@@ -86,7 +100,7 @@
                     <h5 class="mr-auto mb-0">Transactions</h5>
                     {{if lt .NumFundingTxns .KnownFundingTxns}}
                     <div div class="d-flex flex-wrap-reverse align-items-center justify-content-end">
-                        <span class="fs12 nowrap text-right">funding transactions {{intComma (add .Offset 1)}} &mdash; {{intComma (add .Offset .NumFundingTxns)}} of {{intComma .KnownFundingTxns}}
+                        <span class="fs12 nowrap text-right"> {{intComma (add .Offset 1)}} &mdash; {{intComma (add .Offset .NumFundingTxns)}} transactions of {{intComma .KnownFundingTxns}}
```

Comment:
OK, but there is reason to say "funding transactions" and not just "transactions"
IMO it's going to be confusing to see a table with 3 rows (say 2 paying to the address and 1 spending), but this text saying "1 - 2 transactions of 2"
I think it would be better to say "1 - 2 of 2 transactions paying to this address"

Reply:
nvm.  Please also test with postgresql (full mode)